llvm/test/MC/WebAssembly/lto-set-conditional.ll

```diff
@@ -0,0 +1,13 @@
+; RUN: llc %s -filetype=obj
+
+target triple = "wasm32-unknown-unknown"
+
+; In .lto_set_conditional, the first symbol is renamed to the second symbol, so
+; the first symbol does not exist anymore in the file. Object writer should not
+; crash when .lto_set_conditional is present.
+
+module asm ".lto_set_conditional a,a.new"
```

Comment:
Could we wring this in asm directly instead? (i.e could this test be a .s file?)

Reply:
Done